tests/unit/test_dashboards.py

```diff
@@ -191,6 +193,20 @@ def test_dashboard_metadata_validate_misses_tile_content(tmp_path):
     assert "Tile has empty content:" in str(e.value)
 
 
+def test_dashboard_metadata_validate_raises_value_error_for_non_alphanumeric_tile_id(tmp_path):
```

Comment:
Missing test for dashboard name to contain special characters

Reply:
Will move this to a separate PR as dashboard names allow more than alphanumeric characters, dashes and underscore through the UI. I expect that change to be reverted by the Lakeview team